charts/bc-explorer/templates/deploy.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-bc-explorer
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}-bc-explorer
+    spec:
+      serviceAccountName: {{ .Release.Name }}-bc-explorer
+      containers:
+      - name: viewer
+        image: {{ .Values.deployment.image }}
+        command:
+        - viewer
+        args:
+        - -v=5
+        - -db={{ .Values.runtimeArgs.explorerDB }}
+        - -dsn={{ template "pg-svc" .}}
+        ports:
+        - containerPort: 9998
+      - name: listener
+        image:  {{ .Values.deployment.image }}
+        command:
+        - listener
+        args:
+        - -v=5
+        - -injector={{ .Values.runtimeArgs.explorerDB }}
+        - -dsn={{ template "pg-svc" . }}
+        ports:
+        - containerPort: 9999
+      - name: observer
```

Comment:
How about this `observer`? Will observer panic ff current cluster do not have ibp resources?

Reply:
If the current cluster do not have ibp resources, the observer will log error, but no panic
```bash
E0609 14:01:45.412709   26897 reflector.go:138] github.com/IBM-Blockchain/fabric-operator/pkg/generated/informers/externalversions/factory.go:119: Failed to watch *v1beta1.Channel: failed to list *v1beta1.Channel: the server could not find the requested resource (get channels.ibp.com)
...
```

Reply:
🆗 then it will be fine